components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2980,6 +2985,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         let user = this.checkAndBlockUser("getSupportedWorkspaceClasses");
         let selectedClass = await WorkspaceClasses.getConfiguredOrUpgradeFromLegacy(
             user,
+            undefined,
```

Comment:
Why are we not passing the project here?

Reply:
relevant: https://github.com/gitpod-io/gitpod/pull/14535/files#r1017953380

Reply:
We don't have a project context